hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/crypto/HapiCryptoTransfer.java

```diff
@@ -483,6 +500,31 @@ protected Consumer<TransactionBody.Builder> opBodyDef(final HapiSpec spec) throw
         return builder -> builder.setCryptoTransfer(opBody);
     }
 
+    private void FungibleTokenPartitionMove(final CryptoTransferTransactionBody.Builder b, final HapiSpec spec) {
+        final var extra = (this.PARTITION_TOKEN_OPERATION_MODE == "PartitionMoveWithoutUserSignature")
+                ? appendedPartitionFromTo.get()
+                : appendedPartitionFromTowithSign.get();
+        final var involved = extra.getLeft();
+        final var fromAccount = TxnUtils.asId(involved[0], spec);
+        final var fromPartitionToken = TxnUtils.asTokenId(involved[1], spec);
+        final var toAccount = (this.PARTITION_TOKEN_OPERATION_MODE != "PartitionMoveWithoutUserSignature")
```

Comment:
It might be clearer to modify this line like so 

`final var toAccount = (this.PARTITION_TOKEN_OPERATION_MODE = "PartitionMoveWithUserSignature")`

To avoid the confusion of double negatives. Otherwise LGTM 👍

Reply:
@thomas-swirlds-labs, I have pushed the changes.. Can you please check it. Thanks

Reply:
LGTM @vvairava. Thanks 👍